### ISSUE TITLE
🚧Improve font loader 

### ISSUE THREAD

**Issue by CalvinWilkinson** [KinsonDigital/Velaptor#752]
### Complete The Item Below

- [X] I have updated the title without removing the 🚧 emoji.

### Description

Improve the font loading process.

Currently, the `FontLoader` class loads font using a combination of the font name and the font size.  This is accomplished by building a string that includes the full font file path with the font size in an opinionated way and used as a _**unique key**_ in the font cache.  This needs to be improved for the end user.

The end-user has to know the special syntax required to load a font of a particular size.  This is not ideal and can lead to confusion and a bad developer experience.

Here is the special string syntax requirement:
- `TimesNewRoman|size:12`

Instead of requiring the user to enter a specially formatted string to choose the size, have the user enter an additional font size parameter in the `Load()` method of the loader.

### Solution

This can be done by creating some extension methods so the user can get the ability to use the 2 parameters instead of the single parameter version which requires the special syntax.

1. Create a `static class` named `ContentExtensions`
2. Create an extension method to extend the `FontLoader` type
    - This will take 2 parameters.  Param 1 will be the name of the content and param 2 will be the size of type `uint`
3. Create an extension method to extend the `ILoader<IFont>` type
    - This will take 2 parameters.  Param 1 will be the name of the content and param 2 will be the size of type `uint`

> **Note**
> The implementation of both the extension methods will create the font path and meta-data string to send into the original method.

> **Warning**
> The original `Load()` method implementation for the `FontLoader` class needs to stay in place.  We have no choice but to utilize this meta-data setup due to the `ILoader<T>` interface.  Loading fonts are the only content that requires 2 different pieces of information.

### Acceptance Criteria

- [x] Extension methods created
    - [x] Extension method created for `ILoader<IFont>.Load()` type
    - [x] Extension method created for `FontLoader.Load()` type

### ToDo Items

- [X] Change type labels added to this issue.  Refer to the _**Change Type Labels**_ section below.
- [X] Priority label added to this issue.  Refer to the _**Priority Type Labels**_ section below.
- [x] Issue linked to the correct milestone _(if applicable)_.

### Issue Dependencies

- KinsonDigital/VelaptorDocs#80
    - This needs to be done first to help develop **_Velaptor_** features and release docs at the same time

### Additional Information:

**_<details closed><summary>Change Type Labels</summary>_**

| Change Type         | Label                |
|---------------------|----------------------|
| Bug Fixes           | `🐛bug` |
| Breaking Changes    | `🧨breaking changes` |
| New Feature         | `✨new feature` |
| CICD Changes        | `♻️cicd` |
| Config Changes      | `⚙️config` |
| Performance Improvements | `🏎️performance` |
| Code Doc Changes    | `🗒️documentation/code` |
| Product Doc Changes | `📝documentation/product` |
</details>

**_<details closed><summary>Priority Type Labels</summary>_**

| Priority Type       | Label             |
|---------------------|-------------------|
| Low Priority        | `low priority`    |
| Medium Priority     | `medium priority` |
| High Priority       | `high priority`   |
</details>

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct.
